cmake/scripts/freebsd/ArchSetup.cmake

```diff
@@ -1,6 +1,6 @@
 set(ARCH_DEFINES -DTARGET_POSIX -DTARGET_FREEBSD)
 set(SYSTEM_DEFINES -D__STDC_CONSTANT_MACROS -D_LARGEFILE64_SOURCE
-                   -D_FILE_OFFSET_BITS=64)
+                   -D_FILE_OFFSET_BITS=64 -DHAS_OSS)
```

Comment:
is this not set by buildsystem probe?

Reply:
no, since there is no FindOSS module (anymore), this should be added